sideline-flymake.el

```diff
@@ -51,6 +51,11 @@
   :group 'tool
   :link '(url-link :tag "Repository" "https://github.com/emacs-sideline/sideline-flymake"))
 
+(defcustom sideline-flymake-display-errors-whole-line nil
+  "Weather to display errors for the whole line."
+  :type 'boolean
+  :group 'sideline-flymake)
```

Comment:
How about a symbol instead of a boolean? `'point` or `'line`, etc. This way we can expand this when it's needed.

```elisp
(defcustom sideline-flymake-display-mode 'point
  "..."
  :type '(choice (const line)
                 (const point))
  :group 'sideline-flymake)
```

WDYT?

Reply:
That's way cleaner indeed